vignettes/tidytranscriptomics.Rmd

```diff
@@ -964,16 +1023,40 @@ pbmc_bulk %>%
   tidybulk::scale_abundance()
 ```
 
+## tidyseurat
```

Comment:
Are we including tidyseurat in the Bioc workshop?

Reply:
No XD